firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -12,8 +14,7 @@ class TestAboutPrivateBrowsing(FirefoxTestCase):
 
     def setUp(self):
         FirefoxTestCase.setUp(self)
-        self.pb_url = self.marionette.absolute_url('private_browsing/'
-                                                   'about.html?')
+        self.pb_url = self.marionette.absolute_url('private_browsing/about.html?')
```

Comment:
Please don't keep a lot of unrelated changes around. It makes reviewing a lot harder.